docs/01-app/02-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -248,6 +248,67 @@ export default function Page() {
 
 Navigating to the `/mdx-page` route should display your rendered MDX page.
 
+<AppOnly>
+
+### Using dynamic imports
+
+You can also dynamically import MDX in your pages.
+
+Create a new dynamic page within the `/app` directory and add MDX files wherever you'd like:
+
+```txt
+  my-project
+  ├── app
+  │   └── blog
+  │       └── [slug]
+  │           └── page.(tsx/js)
+  ├── markdown
+  │   ├── welcome.(mdx/md)
+      └── about.(mdx/md)
+  |── mdx-components.(tsx/js)
+  └── package.json
+```
```

Comment:
@samcx should we make a Figma diagram for this structure? Happy to help 😄

Reply:
That would be great!